README.md

```diff
@@ -28,6 +28,61 @@ let alexa = new Alexa.Router()
 Once you intialize the router, you can either configure `actions` or `dispatch` a HTTP request to be
 routed to the actions you have configured.
 
+### `Router`
+
+#### API
+
+`new Alexa.Router(config)`
+
+#### config
+*Required* <br>
+Type: `object`
+
+
+`config.appId` <br>
+*Required*  <br>
+Type: `String[]`
+
+Your application ID or an array with many
+
+`config.routeIntentOnly` <br>
+*Optional*  <br>
+Type: `Boolean` <br>
+Default: `true`
+
+Try to route `IntentRequest` only
+
+`config.verifySignature` <br>
+*Optional*  <br>
+Type: `Boolean` <br>
+Default: `true`
+
+Verifies the incoming request against a valid Amazon's signature to prevent request forgery
```

Comment:
"a valid Amazon"

"...request forgery. Amazon [requires verification](https://developer.amazon.com/public/solutions/alexa/alexa-skills-kit/docs/developing-an-alexa-skill-as-a-web-service#Verifying) as part of the skill submission process"